win32_event_log/datadog_checks/win32_event_log/check.py

```diff
@@ -400,7 +416,7 @@ def get_session_struct(self):
         if auth_type not in self.LOGIN_FLAGS:
             raise ConfigurationError('Invalid `auth_type`, must be one of: {}'.format(', '.join(self.LOGIN_FLAGS)))
 
-        user = self.instance.get('user')
+        user = self.instance.get('user', self.instance.get('username'))
```

Comment:
This is for a totally different auth method, so I don't think we should conflate the 2

Reply:
But you cannot use both authentication methods at the same time. I find super redundant and confusing to have both `user` and `username` present on the file and having to discern when to use each.